tgui/packages/tgui/interfaces/Hypospray.tsx

```diff
@@ -0,0 +1,66 @@
+import { BooleanLike } from 'common/react';
+
+import { useBackend } from '../backend';
+import { Box, Button } from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  InjectMode: BooleanLike;
+  CurrentLabel: string;
+  CurrentTag: string;
+  TransferAmount: number;
+  IsAdvanced: boolean;
+};
+
+export const Hypospray = (props) => {
+  const { act, data } = useBackend<Data>();
+  const { InjectMode, CurrentLabel, CurrentTag, TransferAmount, IsAdvanced } =
+    data;
+  return (
+    <Window width={300} height={375}>
+      <Box>
+        <Button m={2} onClick={() => act('ActivateAutolabeler')}>
+          Activate Autolabeler
+        </Button>
+        <Box ml={2}>
+          <b>Current label: </b>
+          {CurrentLabel}
```

Comment:
why not use bold argument here?

Reply:
Because that would also make the test/number also be in bold, which I don't want.
![image](https://github.com/tgstation/TerraGov-Marine-Corps/assets/66163761/ed011af2-cf13-4d5d-b96e-4e51c1cc7470)

Reply:
shouldve used labeledlist